lib/mix/lib/mix/task.ex

```diff
@@ -124,9 +124,10 @@ defmodule Mix.Task do
   """
   @spec moduledoc(task_module) :: String.t() | nil
```

Comment:
According to the typespec we always return `nil`. Maybe we could get away with returning `nil` for `false` as well?

Reply:
Good catch, I should've looked at the typespec.